pkg/extensions/config/events/config.go

```diff
@@ -0,0 +1,55 @@
+package events
+
+import (
+	"time"
+)
+
+type SinkType string
+
+func (s SinkType) String() string {
+	return string(s)
+}
+
+const (
+	HTTP SinkType = "http"
```

Comment:
Just calling this out.
Currently, these are the two types of sinks supported.
Should be easy to add more if needed.